### ISSUE TITLE
GTlab crashes when calling GtCalculators.findGtTask() from an external script file

### ISSUE THREAD

**Issue by marvinoe21** [dlr-gtlab/python-module#568]
When calling the GtCalculators.findGtTask() function from an external script file, GTlab crashes unexpectedly.

## Steps to Reproduce

1. Start GTlab version 2.0.10 and ensure that the GTlab Python Module 1.6.0 is loaded.
2. Create a directory named `python_scripts` in the same directory as `GTlab.exe`.
3. In the `python_scripts` directory, create a Python script named `external_script.py` that calls `GtCalculators.findGtTask()`. Here is an example code to copy:
```python
from GtCalculators import findGtTask

def external_find_task(task_name: str):
    task = findGtTask(task_name)
```
4. In GTlab create a Simple Task named `Task`.
5. Create a Python Task and copy the following code into it:

```py
import os
import sys

external_scripts_dir = os.path.join(os.getcwd(), 'python_scripts')
sys.path.append(external_scripts_dir)

# Load the external module
from external_script import external_find_task
del sys.modules['external_script']
from external_script import external_find_task

external_find_task('Task')
```
6. Run the script and observe that GTlab crashes.

**Comment by rainman110** [comment]
@marvinoe21 Is this specific to the python module in version 1.6.0, i.e. due to the changes in the findGtTask function?

**Comment by rainman110** [comment]
> # Load the external module
>  from external_script import external_find_task
>  del sys.modules['external_script']
>  from external_script import external_find_task

What is is, that you are trying to do here? Why do you import twice? Is this required to reproduce the issue?

**Comment by rainman110** [comment]
Fixed with commit 43f2e2c
